src/shared/components/results/RenderColumnInCard.tsx

```diff
@@ -0,0 +1,26 @@
+import { ColumnConfiguration } from '../../types/columnConfiguration';
+
+type RenderColumnInCardProps<T, R extends Record<string, unknown>> = {
+  type: T;
+  data: R;
+  columnConfig: ColumnConfiguration<T, R>;
+};
+
+const RenderColumnInCard = <T, R extends Record<string, unknown>>(
+  props: RenderColumnInCardProps<T, R>
+) => {
+  const { type, data, columnConfig } = props;
+  const rendered = columnConfig.get(type)?.render(data);
+  return rendered ? (
+    <>
+      <span className="result-card__info-bit">
+        {columnConfig.get(type)?.label}
+        {': '}
+        {rendered}
+      </span>{' '}
+      {' · '}
```

Comment:
Maybe this separator should be done in CSS as a pseudo element `:after:not(:last-child)` or something like that.
The issue now is that the last instance of it will have a dot afterwards which looks a bit strange